alpine/workfile.py

```diff
@@ -494,11 +498,18 @@ def wait_until_finished(self, process_id, verbose=False, query_time=10, timeout=
                         " It now has status '{2}'.".format(process_id, timeout, stop_status))
 
                 if verbose:
-                    print("\rWorkflow in progress for ~{0:.2f} seconds.".format(wait_total)),
+                    print("\rWorkflow in progress for ~{0:.1f} seconds.".format(wait_total)),
```

Comment:
please update print to self.logger.debug so you can see also see the output print while you debugging and we can easily switch(enable/disable) debug info off in production environment